File: src/test/resources/ru/vyarus/gradle/plugin/animalsniffer/java/invalid/Sample.java
package invalid;

import org.slf4j.LoggerFactory;

import java.nio.file.Paths;

public class Sample {

    public static void main(String[] args) {
        // method added in 1.7
        Boolean.compare(true, true);
    }

    public void someth() {
        // class added in 1.7
        Paths.get("/tmp");
        LoggerFactory.getLogger(Sample.class);
    }
}

File: src/test/resources/ru/vyarus/gradle/plugin/animalsniffer/scala/invalid/Sample.scala
package invalid

import org.slf4j.LoggerFactory

import java.nio.file.Paths
import java.lang.Boolean

object Sample {

    def main(args: Array[String]): Unit = {
        // method added in 1.7
        Boolean.compare(true, true)
    }

    def someth(): Unit = {
        // class added in 1.7
        Paths.get("/tmp")
        LoggerFactory.getLogger("sample logger")
    }
}